serverless/verify-backup/src/server.ts

```diff
@@ -1,14 +1,33 @@
 import express, { Request, Response } from 'express'
 import bodyParser from 'body-parser'
+import { spawn } from 'child_process'
```

Comment:
Another dev only issue. 

Environment variables in `.env` are not loaded on the first POST request. `config` is only imported when `npm run decrypt-dump` is called. Consequently, `dotenv.config()` is also only called at that time. As such when the child process is spawned, environment variables (e.g. `CRONITOR_CODE`) is not set/available.

Reply:
Noted, I have imported `dotenv` in `server.ts` instead